lib/administrate/generator_helpers.rb

```diff
@@ -0,0 +1,7 @@
+module Administrate
+  module GeneratorHelpers
+    def call_generator(generator, *args)
+      Rails::Generators.invoke(generator, args, behavior: behavior)
```

Comment:
normally I'd agree, but that would defeat the purpose of extracting this method. I wanted to automatically include the parent's behavior, instead of passing it in explicitly.